test/TargetFunctions.sol

```diff
@@ -255,6 +276,34 @@ abstract contract TargetFunctions is Properties {
         // --- Ghost data after ---
         last_action = LastAction.DONATE;
         __totalAssetAfter = woeth.totalAssets();
+        __oeth_balanace_of_woeth = oeth.balanceOf(address(woeth));
+        __hardAssets = woeth.hardAssets();
+        __yieldAssets = woeth.yieldAssets();
+        __yieldEnd = woeth.yieldEnd();
+    }
+
+    /// @notice Handle pass time on chain
+    /// @param _amount Amount of time to pass. 1 Day is maximum, since that is also the 
+    ///        maximum yield time 
+    function handler_pass_time(uint88 _amount) public {
+        uint256 MAX_YIELD_TIME = 1 days;
```

Comment:
Maybe could you move this to `HelperAssumption.sol` contract?

Reply:
agreed: https://github.com/OriginProtocol/woeth-fuzzing-tests/pull/1/commits/94d37cc37cb61181c1769165b6d8240af56d489d